plugins/module_utils/backup.py

```diff
@@ -76,29 +78,37 @@ def mvs_file_backup(dsn, bk_dsn=None, tmphlq=None):
             bk_dsn = extract_dsname(dsn) + "({0})".format(temp_member_name())
 
         bk_dsn = _validate_data_set_name(bk_dsn).upper()
-        response = datasets._copy(dsn, bk_dsn)
-        if response.rc != 0:
+        try:
+            rc = datasets.copy(dsn, bk_dsn)
+        except exceptions.ZOAUException as copy_exception:
             raise BackupError(
                 "Unable to backup {0} to {1}".format(dsn, bk_dsn),
-                rc=response.rc,
-                stdout=response.stdout_response,
-                stderr=response.stderr_response
+                rc=copy_exception.response.rc,
+                stdout=copy_exception.response.stdout_response,
+                stderr=copy_exception.response.stderr_response
+            )
+        if rc != 0:
```

Comment:
Did you want to remove this snippet since its being handled by the updated except:
```
        if rc != 0:
            raise BackupError(
                "Unable to backup {0} to {1}".format(dsn, bk_dsn),
                rc=rc
            )
```

`rc`  seems to be local to the try/except as well and rc is now the zoau data set object, which you probably don't need anymore.

Reply:
yeah that was the intention. removed it now